lib/howl/keymap.moon

```diff
@@ -16,7 +16,9 @@
     tab:              'editor-smart-tab'
     shift_tab:        'editor-smart-back-tab'
     backspace:        'editor-delete-back'
+    ctrl_backspace:   'editor-delete-back-word'
```

Comment:
I think it'd be bound to Option+Backspace/Delete on the Mac - but I noticed this comment re option key in the keymap: https://github.com/howl-editor/howl/blob/master/lib/howl/keymap.moon#L152